README.md

```diff
@@ -6,7 +6,106 @@ The Microsoft Graph Communications API allows developers to programmatically int
 
 - View the **[changelog](changelog.md)**
 - **[Review the documentation](https://microsoftgraph.github.io/microsoft-graph-comms-samples/docs/)** to understand the concepts behind using our SDK (which is also used by the samples).
-- Get started quickly with our **[samples](https://github.com/microsoftgraph/microsoft-graph-comms-samples/tree/master/Samples)**.
+
+## Prerequisites
+
+* Visual Studio. You can download the community version [here](http://www.visualstudio.com) for free.
+* Mirosoft Azure Subscription (If you do not already have a subscription, you can register for a <a href="https://azure.microsoft.com/en-us/free/" target="_blank">free account</a>)
+* [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest)
+* [Azure PowerShell](https://docs.microsoft.com/en-us/powershell/azure/install-azurerm-ps?view=azurermps-6.8.1)
+* [Skype for Business Online Connector module](https://docs.microsoft.com/en-us/office365/enterprise/powershell/manage-skype-for-business-online-with-office-365-powershell)
+* Install .Net Framework 4.7.1. Some samples will not build if you do not install this.
```

Comment:
Remove the word "Install"
Provide a link to download [.NET Framework 4.7.1](https://dotnet.microsoft.com/download/dotnet-framework/net471)